### PR TITLE
master 🤝 dhyey's pile of wackness 

### PR DESCRIPTION
**Pull request by DhyeyLalseta**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+repos:
```
Comment (RatikKapoor) [inline .pre-commit-config.yaml:1]: Love pre-commit ❤️

File: .style.yapf [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[style]
```
Comment (RatikKapoor) [inline .style.yapf:1]: Awesome job on YAPF!

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,48 @@
+# For more information, please refer to https://aka.ms/vscode-docker-python
```
Comment (RatikKapoor) [inline Dockerfile:1]: 🐳

File: README.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+# Relectric Systems
```
Comment (RatikKapoor) [inline README.md:1]: I'll probably clean up the README later once I get more actions in but this is good for now

File: systems/abstract.py [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+        Args:
+            socket (Socket)
+
+        Returns:
+            bool: True if connection granted
+        """
+        socket.send(b'')
+        ready_ping = socket.recv_multipart()
+        self.is_vibe_checked = b'' in ready_ping
```
Comment (RatikKapoor) [inline systems/abstract.py:23]: Lol, we should use proper names from now on

**Review by RatikKapoor — DISMISSED** [review]
🤝 , next time let's use semantic commit message names and also apply that same standard to PRs. Apart from that great job!! 🚙

File: .pre-commit-config.yaml (hunk 1 above)
Comment (rjb75) [reply .pre-commit-config.yaml:1]: Pre-commit = good times

**Review by rjb75 — APPROVED** [review]
Ø MQ is very cool!

**Review by RatikKapoor — APPROVED** [review]

File: pyproject.toml [hunk 6]
```diff
@@ -0,0 +1,26 @@
+[tool.poetry]
+name = "systems"
+version = "0.1.0"
+description = "Systems code for Relectric car team."
+authors = ["Relectric Car Team <info@teamrelectric.ca>"]
+license = "MIT"
+
+[tool.poetry.dependencies]
+python = "^3.7"
```
Comment (elizetran) [inline pyproject.toml:9]: python --version
2.7.16 😪

**Review by elizetran — APPROVED** [review]
Really great job on this!

**Review by RatikKapoor — APPROVED** [review]
:shipit:

**Pull request merged**